### PR TITLE
implement GET email message template endpoint

### PR DESCRIPTION
**Pull request by eugene-belkovich**

### CODE REVIEW

File: src/routes/notification/template/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import TemplateController from '../../../controllers/TemplateController';
+
+const router = express.Router();
+
+router.get('/', TemplateController.getTemplates);
+router.get('/:id', TemplateController.getTemplate);
```
Comment (magonzalez) [inline src/routes/notification/template/index.js]: This doesn't exist. The code doesn't run.

File: src/routes/notification/template/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+export default router;
```
Comment (magonzalez) [inline src/routes/notification/template/index.js]: These routes need to be registered in the src/makeroutes.js file otherwise they are not accessible.

Comment (eugene-belkovich) [reply src/routes/notification/template/index.js]: I checked it, these routes works.

**Comment by eugene-belkovich** [comment]
Also, do I need to rename TemplateController, TemplateService and etc to NotificationTemplateController?

File: src/makeRoutes.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
 import userRouter from './routes/user';
 import userAuthRouter from './routes/user/auth';
+import NotificationTemplateController from './controllers/NotificationTemplateController';
```
Comment (magonzalez) [inline src/makeRoutes.js]: Should import the router file for NotificationTemplates just like we are doing for Users, not the controller like you're doing.

**Pull request merged**